AnkiDroid/src/main/java/com/ichi2/libanki/Decks.java

```diff
@@ -574,7 +574,8 @@ public String _ensureParents(String name) {
         if (path.size() < 2) {
             return name;
         }
-        for(String p : path.subList(0, path.size() - 1)) {
+        for(int i=0; i<path.size() - 1; i++) {
```

Comment:
AnkiDroid coding standards - whitespace around operators so they stand out visually

```suggestion
        for(int i = 0; i < path.size() - 1; i++) {
```